internal/prometheus/job_metrics.go

```diff
@@ -54,35 +54,37 @@ var (
 		Subsystem: workerSubsystem,
 		Help:      "Duration a job spends on the queue.",
 		Buckets:   []float64{.1, .2, .5, 1, 2, 4, 8, 16, 32, 40, 48, 64, 96, 128, 160, 192, 224, 256, 320, 382, 448, 512, 640, 768, 896, 1024, 1280, 1536, 1792, 2049},
-	}, []string{"type"})
+	}, []string{"type", "tenant"})
 )
 
-func EnqueueJobMetrics(jobType string) {
-	PendingJobs.WithLabelValues(jobType).Inc()
+func EnqueueJobMetrics(jobType, channel string) {
+	tenant := channel[4:]
+	PendingJobs.WithLabelValues(jobType, tenant).Inc()
 }
 
-func DequeueJobMetrics(queued time.Time, started time.Time, jobType string) {
+func DequeueJobMetrics(queued time.Time, started time.Time, jobType, tenant string) {
 	if !started.IsZero() && !queued.IsZero() {
 		diff := started.Sub(queued).Seconds()
-		JobWaitDuration.WithLabelValues(jobType).Observe(diff)
-		PendingJobs.WithLabelValues(jobType).Dec()
-		RunningJobs.WithLabelValues(jobType).Inc()
+		JobWaitDuration.WithLabelValues(jobType, tenant).Observe(diff)
+		PendingJobs.WithLabelValues(jobType, tenant).Dec()
+		RunningJobs.WithLabelValues(jobType, tenant).Inc()
 	}
 }
 
-func CancelJobMetrics(started time.Time, jobType string) {
+func CancelJobMetrics(started time.Time, jobType string, channel string) {
```

Comment:
This is a bit of a nitpick. It was a little confusing at first because in some cases we are passing in the `channel` and then getting the sub slice (like this case) and then in other cases we are passing the `tenant`.

Maybe, for consistency, we could take the tenant as an argument in all these functions?